CODEOWNERS

```diff
@@ -498,12 +498,12 @@
 /itests/org.openhab.binding.astro.tests/ @gerrieg
 /itests/org.openhab.binding.avmfritz.tests/ @cweitkamp
 /itests/org.openhab.binding.feed.tests/ @openhab/add-ons-maintainers
+/itests/org.openhab.binding.homeassistant.tests/ @antroids
```

Comment:
Not strictly related to this PR, better in a seperate PR as if we ever revert we can get issues.